candies/cli/command.py

```diff
@@ -0,0 +1,68 @@
+from typing import Callable, Optional, Union
+
+
+class Command:
+    """A command of a CLI.
+
+    Attributes:
+        func: A function to be executed when the command is invoked.
+        name: A name of the command
+            (`func.__name__` if not explicitly specified).
+        description: A description of the command
+            (`func.__doc__` if not explicitly specified).
+        subcommands: A dictionary that maps a name of a subcommand
+            to the subcommand itself.
+
+    Examples:
+        @cli
```

Comment:
Not really. In my opinion, this would be too much for a single docstring. I believe that we need a separate `examples` folder with a comprehensive list of examples (including sub-subcommands).

Reply:
Agreed.